CommunityProject/Assets/_Game/Scripts/Runtime/UI/ViewBuildNumber.cs

```diff
@@ -0,0 +1,35 @@
+using BoundfoxStudios.CommunityProject.Build.BuildManifest;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using System;
+
+namespace BoundfoxStudios.CommunityProject
+{
+    public class ViewBuildNumber : MonoBehaviour
+    {
+        private TextMeshProUGUI BuildNumberText;
```

Comment:
```suggestion
        private TextMeshProUGUI _buildNumberText;
```

Das kann leider der Linter noch nicht automatisch prüfen, da bin ich noch am Werkeln, wie das. geht. Bzgl. der Benamung siehe auch: https://github.com/BoundfoxStudios/community-project/blob/develop/docs/contributing/technical/coding-conventions.md#groß-kleinschreibung